### PR TITLE
Niko/user and post tests

### PR DESCRIPTION
**Pull request by niko79542**
updates user and post model tests

### CODE REVIEW

File: src/controllers/graphql/schemas/index.js [hunk 1]
```diff
@@ -1,13 +1,81 @@
...
+      return user.getPosts();
+    }
+  },
+  Post: {
+    user(post) {
+      return post.getUser();
+    }
+  }
+};
+
```
Comment (jay-ithiel) [inline src/controllers/graphql/schemas/index.js]: We should split these up into their selective files, import the files into here, merge and export. Like the way it was before - each model has its own file, import the files, merge and export

File: src/controllers/graphql/schemas/post/post.js [hunk 2]
```diff
@@ -78,6 +78,11 @@ const resolvers = {
     votePost: async (_, args) => { // { accessToken: 'kjhDG5THrg', author: 'steemit', permlink: 'firstpost', upvote: 1, vote_percent: 50 }
       return 'success'
     }
+  },
+  Post: {
+    user(post) {
+      return post.getUser();
+    }
   }
```
Comment (jay-ithiel) [inline src/controllers/graphql/schemas/post/post.js:90]: Seems lines 82-86 are the same as lines 25-29 in `src/controllers/graphql/schemas/index.js`
Are they doing the same thing?

File: src/controllers/graphql/schemas/user/user.js [hunk 3]
```diff
@@ -27,6 +34,11 @@ const resolvers = {
       return "https://steemconnect.com/oauth2/authorize?client_id";
     },
   },
+  User: {
+    posts(user) {
+      return user.getPosts();
+    }
+  }
 };
```
Comment (jay-ithiel) [inline src/controllers/graphql/schemas/user/user.js:42]: Same here - looks like this is duplicate code?

**Comment by jay-ithiel** [comment]
Your `niko/mount-graphql-schema` branch was merged into `master`, so there are merge conflicts in your other pull requests now because `master` is 1 commit ahead of those other branches. To resolve:

```
git co master
git pull origin master
git co niko/working-branch-name
git merge master
```
resolve conflicts
```
git add .
git commit -m "merge master"
git push origin niko/working-branch-name
```

**Comment by niko79542** [comment]
Removed duplicate code

**Pull request merged**